packages/commonwealth/server/workers/sitemap/runner.ts

```diff
@@ -0,0 +1,15 @@
+import { createAsyncWriterS3 } from './createAsyncWriter';
+import { createDatabasePaginatorDefault } from './createDatabasePaginator';
+import { createSitemapGenerator } from './createSitemapGenerator';
+
+async function doExec() {
+  const writer = createAsyncWriterS3();
+  const paginator = createDatabasePaginatorDefault();
+
+  await createSitemapGenerator(writer, [
+    paginator.threads,
+    paginator.profiles,
+  ]).exec();
+}
+
+doExec().catch(console.error);
```

Comment:
Please use the `logging` utility from `@hicommonwealth/logging`:
```
import { logger } from '@hicommonwealth/logging`

const log = logger(__filename);
```

We should `log.fatal('msg here', err);`. This ensures the error will be reported to Rollbar if it occurs in a production or Rollbar enabled environment.

Reply:
Thanks. I'll fix this.